crates/router/src/connector/expresscheckout.rs

```diff
@@ -0,0 +1,618 @@
+    ) -> CustomResult<String, errors::ConnectorError> {
+        let url = format!(
+            "{}{}{}",
+            self.base_url(connectors),
+            "orders/",
+            req.payment_id.clone()
+        );
+        Ok(url)
+    }
+
+    fn build_request(
+        &self,
+        req: &types::PaymentsSyncRouterData,
+        connectors: &settings::Connectors,
+    ) -> CustomResult<Option<services::Request>, errors::ConnectorError> {
+        let body = services::RequestBuilder::new()
+            .method(services::Method::Get)
+            .url(&types::PaymentsSyncType::get_url(self, req, connectors)?)
+            .headers(types::PaymentsSyncType::get_headers(self, req, connectors)?)
+            .build();
+        Ok(Some(body))
+    }
+
+    fn get_error_response(
+        &self,
+        res: Response,
+    ) -> CustomResult<ErrorResponse, errors::ConnectorError> {
+        self.build_error_response(res)
+    }
+
+    fn handle_response(
+        &self,
+        data: &types::PaymentsSyncRouterData,
+        res: Response,
+    ) -> CustomResult<types::PaymentsSyncRouterData, errors::ConnectorError> {
+        logger::debug!(payment_sync_response=?res);
+        let response: expresscheckout::ExpresscheckoutPaymentsResponse = res
+            .response
+            .parse_struct("expresscheckout PaymentsResponse")
+            .change_context(errors::ConnectorError::ResponseDeserializationFailed)?;
+        Ok(types::RouterData::try_from(types::ResponseRouterData {
+            response,
+            data: data.clone(),
+            http_code: res.status_code,
+        })
+        .change_context(errors::ConnectorError::ResponseHandlingFailed)?)
+    }
+}
+
+impl api::PaymentCapture for Expresscheckout {}
+impl ConnectorIntegration<api::Capture, types::PaymentsCaptureData, types::PaymentsResponseData>
+    for Expresscheckout
+{
+    fn get_headers(
+        &self,
+        req: &types::PaymentsCaptureRouterData,
+        connectors: &settings::Connectors,
+    ) -> CustomResult<Vec<(String, String)>, errors::ConnectorError> {
+        self.build_headers(req, connectors)
+    }
+
+    fn get_content_type(&self) -> &'static str {
+        self.common_get_content_type()
+    }
+
+    fn get_url(
+        &self,
+        req: &types::PaymentsCaptureRouterData,
+        connectors: &settings::Connectors,
+    ) -> CustomResult<String, errors::ConnectorError> {
+        let res = req
+            .response
+            .as_ref()
+            .map_err(|_| errors::ConnectorError::FailedToObtainIntegrationUrl)?;
+        if let types::PaymentsResponseData::TransactionResponse { resource_id, .. } = res {
+            if let types::ResponseId::ConnectorTransactionId(r) = resource_id {
+                Ok(format!(
+                    "{}{}{}{}",
+                    self.base_url(connectors),
+                    "v2/txns/",
+                    r,
+                    "/capture/"
+                ))
```

Comment:
Literals can be inlined
```suggestion
Ok(format!("{}v2/txns/{}/capture/",
                    self.base_url(connectors),
                    r
                ))
```
It would be good if this is used in all the get_url functions